libs/src/services/creatorNode/index.js

```diff
@@ -759,7 +759,11 @@ class CreatorNode {
             if (!total) total = progressEvent.total
             console.info(`Upload in progress: ${progressEvent.loaded} / ${total}`)
             onProgress(progressEvent.loaded, total)
-          }
+          },
+          // Set content length headers (only applicable in server/node environments).
```

Comment:
what do you mean that this is only applicable in server/node envs? does client not apply these headers if this function is called?

also when you say that this track upload currently fails in node env, is that basically only maddog / test runs?

Reply:
These aren't headers, but request config. see https://github.com/axios/axios#request-config

It's not just test environments. 3p apps/api partners can consume this method/libs server-side and fail to upload large tracks successfully. (e.g. youtube migrator)

Reply:
ahh i see the docs specify this is only applicable in node env

and right, i didn't think about all the 3p apps/api